contracts/staking/StakingBase.sol

```diff
@@ -213,23 +259,21 @@ abstract contract StakingBase is ERC721TokenReceiver {
     }
 
     event ServiceStaked(uint256 epoch, uint256 indexed serviceId, address indexed owner, address indexed multisig,
-        uint256[] nonces);
+        uint256[] nonces, uint256 rewardDistributionInfo);
     event Checkpoint(uint256 indexed epoch, uint256 availableRewards, uint256[] serviceIds, uint256[] rewards,
         uint256 epochLength);
     event ServiceUnstaked(uint256 epoch, uint256 indexed serviceId, address indexed owner, address indexed multisig,
-        uint256[] nonces, uint256 reward, uint256 availableRewards);
-    event ServiceForceUnstaked(uint256 epoch, uint256 indexed serviceId, address indexed owner, address indexed multisig,
-        uint256[] nonces, uint256 reward, uint256 availableRewards);
```

Comment:
This event is not needed, better to state in a `ServiceUnstaked()` event if unstake was forced or not.